Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -251,68 +275,106 @@ private static Encoding GetEncoding(Destination dest)
         }
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/56824f42a881d74709954eeb239fb474ee075540/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/SingleFileTransfer.cs#L274
Please add some kind of error message here.